app/src/test/java/com/google/android/ground/ui/offlineareas/selector/DownloadProgressDialogTest.kt

```diff
@@ -23,21 +23,33 @@ import androidx.compose.ui.test.onNodeWithText
 import androidx.compose.ui.test.performClick
 import com.google.android.ground.BaseHiltTest
 import com.google.android.ground.R
+import com.google.android.ground.repository.OfflineAreaRepository
+import com.google.android.ground.ui.map.Bounds
 import dagger.hilt.android.testing.HiltAndroidTest
 import javax.inject.Inject
+import junit.framework.Assert.assertFalse
 import kotlin.test.Test
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.test.advanceUntilIdle
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotEquals
+import org.junit.Assert.assertNull
 import org.junit.Assert.assertTrue
 import org.junit.Rule
 import org.junit.runner.RunWith
+import org.mockito.kotlin.whenever
 import org.robolectric.RobolectricTestRunner
 
+@OptIn(ExperimentalCoroutinesApi::class)
 @HiltAndroidTest
 @RunWith(RobolectricTestRunner::class)
 class DownloadProgressDialogTest : BaseHiltTest() {
```

Comment:
This test is only meant to be used for the compose UI. What you want to test should be done in `OfflineAreaSelectorFragmentTest`